operations/boot_orchestration/Options.md

```diff
@@ -20,20 +20,24 @@ Example output:
 
 ```json
 {
+  "bss_read_timeout": 20,
   "cfs_read_timeout": 20,
   "cleanup_completed_session_ttl": "7d",
   "clear_stage": false,
   "component_actual_state_ttl": "4h",
   "default_retry_policy": 3,
   "disable_components_on_completion": true,
   "discovery_frequency": 300,
+  "hsm_read_timeout": 20,
   "ims_errors_fatal": false,
   "ims_images_must_exist": false,
+  "ims_read_timeout": 20,
```

Comment:
IMS is single-threaded for the http server. Some operations may take a while (delete of large images for example) and this may not be a long enough time out for all situations.

This is just the docs though, so I assume the actual default is set in code so no real need to change here just bringing it up...